src/InputOutput/writers.jl

```diff
@@ -451,16 +451,88 @@ function write_new!(
 end
 
 # write fields
+"""
+    write!(writer::HDF5Writer, field::Fields.Field, name::AbstractString)
+
+Calls `write!` with the `axes` of `field` as an extra positional argument, which
+writes an object of type 'Field' and name 'name' to the HDF5 file.
```

Comment:
This docstring describes the implementation detail of the function (calling another method), but I don't think that they are imortant here.

```julia
Write the `field` to the HDF5 in `writer` and assign it the given `name`.
```